src/js/components/sharedComponents/SVGLine.jsx

```diff
@@ -8,25 +8,30 @@ import PropTypes from 'prop-types';
 import { throttle } from 'lodash';
 
 const propTypes = {
-    xScale: PropTypes.func, // function to set line position
+    scale: PropTypes.func, // function to set line position
     text: PropTypes.oneOfType([
         PropTypes.string,
         PropTypes.array
     ]), // text to display with line
-    y1: PropTypes.number, // top of graph
-    y2: PropTypes.number, // bottom of graph
-    xMax: PropTypes.number, // max possible value of x
-    xMin: PropTypes.number, // max possible value of x
-    xValue: PropTypes.number, // actual value of x
+    y1: PropTypes.number, // end of line - vertical
+    y2: PropTypes.number, // start of line - vertical
+    x1: PropTypes.number, // start of line - horizontal
+    x2: PropTypes.number, // end of line - horizontal
+    min: PropTypes.number, // max possible value of x or y
+    max: PropTypes.number, // max possible value of x or y
+    position: PropTypes.number, // value of position
     showTextPosition: PropTypes.string, // show text left, right, and top are valid
     textY: PropTypes.number, // show text at this height
     description: PropTypes.string,
     adjustmentX: PropTypes.number, // adjust x for padding
     textClassname: PropTypes.string,
-    lineClassname: PropTypes.string
+    lineClassname: PropTypes.string,
+    noText: PropTypes.bool, // do not show text,
+    isHorizontal: PropTypes.bool,
```

Comment:
Like the tooltip, we should use an enum to specify this

Reply:
I could see that if we are drawing things other that horizontal or vertical but when there are two options is it not convention to use bool?